powerquery-docs/Connectors/SumTotal.md

```diff
@@ -1,70 +1,65 @@
-﻿---
-title: Power Query SumTotal connector
-description: Provides basic information and prerequisites for the SumTotal connector, and outlines limitations and issues.
-author: bezhan-msft
-ms.service: powerquery
-ms.topic: conceptual
-ms.date: 5/25/2021
-ms.author: bezhan
-LocalizationGroup: reference
----
 
-# SumTotal (Beta)
 
->[!Note]
->The following connector article is provided by SumTotal, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the SumTotal website and use the support channels there.
+---
+
+# SumTotal PowerBI Connector
 
 ## Summary
 
 | Item | Description |
 | ---- | ----------- |
-| Release State | Beta |
+| Release State | General Availability |
 | Products | Power BI (Datasets) |
-| Authentication types | OAuth 2.0 |
-| Function Reference Documentation | - |
+| Authentication types | SumTotal credentials via OAuth2.0 |
+| Function Reference Documentation | [Web.Contents](#https://docs.microsoft.com/en-us/powerquery-m/web-contents) <br> [Table.ToNavigationTable](https://docs.microsoft.com/en-us/power-query/helperfunctions#tabletonavigationtable) <br>[Table.FromRecords](https://docs.microsoft.com/en-us/powerquery-m/table-fromrecords) <br> [Json.Document](https://docs.microsoft.com/en-us/powerquery-m/json-document)|
```

Comment:
We don't need this here as there are no new functions being created other than the main connector function.